Formula/gocryptfs.rb

```diff
@@ -25,13 +25,9 @@ class Gocryptfs < Formula
   end
 
   def install
-    ENV["GOPATH"] = buildpath
-    (buildpath/"src/github.com/rfjakob/gocryptfs").install buildpath.children
-    cd "src/github.com/rfjakob/gocryptfs" do
-      system "./build.bash"
-      bin.install "gocryptfs"
-      prefix.install_metafiles
-    end
+    system "./build.bash"
```

Comment:
Not sure if this will need some extra flags.
Arch Linux adds `export GOFLAGS="-buildmode=pie -trimpath -mod=vendor"`
Ref: https://github.com/archlinux/svntogit-community/blob/packages/gocryptfs/trunk/PKGBUILD#L26

Reply:
Those seem to be set in the script already.